libpod/container_internal.go

```diff
@@ -1281,6 +1281,16 @@ func (c *Container) saveSpec(spec *spec.Spec) error {
 	return nil
 }
 
+// localeToLanguage translates POSIX locale strings to BCP 47 language tags.
+func localeToLanguage(locale string) string {
+	locale = strings.Replace(strings.SplitN(locale, ".", 2)[0], "_", "-", 1)
```

Comment:
(Absolutely non-blocking: This would benefit from a comment, at least saying what is the discarded part supposed to be. OTOH there are tests, so there is enough documentation in that sense.)